build.sbt

```diff
@@ -1,6 +1,9 @@
 import Dependencies._
 import Settings._
 
+resolvers += Resolver.sonatypeRepo("releases")
+
+
```

Comment:
Instead of inlining this into the build.sbt, add this to the list of custom resolvers in [`Settings.commonResolvers`](https://github.com/broadinstitute/cromwell/blob/74edee4c5338d25283c0fcf72ddd016b8bf5c4e1/project/Settings.scala#L17-L21).